openage/convert/dataformat/data_formatter.py

```diff
@@ -184,5 +184,10 @@ def export(self, projectdir, requested_formats):
         # we now invoke the content generation for each generated file
         for gen_file in generate_files:
             file_name, content = gen_file.generate()
-            with projectdir[file_name].open('wb') as outfile:
-                outfile.write(content.encode('utf-8'))
+            if gen_file.format_ in {"csv"}:
+                with open('assets/converted/meta.docx', 'ab') as outfile:
+                    outfile.write(('## ' + file_name + '\n').encode('utf-8'))
+                    outfile.write(content.encode('utf-8'))
```

Comment:
hehe no that's too simple. we generate way more files with this method which are not part of the gamespec and should still be separate. also, the `assets/converted/...` must not be hardcoded.

I would change the `def add_data(self, data_set_pile, prefix=None):` to have a new argument `single_output=None`.

This info is then passed around up to the `generate()` function, which returns another value, which is again the `single_output`. Then you can check if the single output was activated, and perform the write you did here.

Namely not just convert every `csv` to the `meta.docx`, but instead let each generated file specify where it wants to be packed to. And that information is provided by `add_data` already.